### ISSUE TITLE
Error installing MC starter application

### ISSUE THREAD

**Issue by michaelashe** [commercetools/merchant-center-application-kit#2134]
**Describe the Issue**
I am running into the below error while installing the starter application for Merchant center.

Steps followed as per the documentation: https://docs.commercetools.com/custom-applications/getting-started/installing-a-starter-application
1. Checkout code from https://github.com/commercetools/merchant-center-application-kit/tree/main/application-templates
2. Ran the below commands to create a custom app
$ npm install --global @commercetools-frontend/create-mc-app
$ create-mc-app my-new-custom-application-project --template starter

3. Start the custom application
cd my-new-custom-application-project
npm start

Failed to compile
./src/components/main-view/main-view.mod.css (./node_modules/css-loader/dist/cjs.js??ref--8-1!./node_modules/postcss-loader/dist/cjs.js??ref--8-2!./src/components/main-view/main-view.mod.css)
Error: Cannot find module '@commercetools-frontend/mc-scripts/src/config/create-postcss-config'

npm version : 6.14.4
node version: 10.19.0

Also attempted the process using yarn instead of npm and received the same error:

Failed to compile.

./src/components/main-view/main-view.mod.css (./node_modules/css-loader/dist/cjs.js??ref--8-1!./node_modules/@commercetools-frontend/mc-scripts/node_modules/postcss-loader/dist/cjs.js??ref--8-2!./src/components/main-view/main-view.mod.css)
Error: Cannot find module '@commercetools-frontend/mc-scripts/src/config/create-postcss-config'
^Cdemo@demo:~/Documents/CT/merchant-center-application-kit/my-new-custom-application-project$ yarn build
yarn run v1.22.4
$ mc-scripts build
Creating an optimized production build...
clean-webpack-plugin: pausing due to webpack errors
Failed to compile.

./src/components/main-view/main-view.mod.css
Error: Cannot find module '@commercetools-frontend/mc-scripts/src/config/create-postcss-config'

error Command failed with exit code 1.

(SUPPORT-11887)

**Comment by emmenko** [comment]
Hi, it looks to me that you have checked out this repository. Don't do that, unless you want to contribute to it 😉

The `@commercetools-frontend/create-mc-app` lets you install the template starter as a new project/folder, similarly to other frameworks (CRA, etc).

<img width="965" alt="image" src="https://user-images.githubusercontent.com/1110551/114203971-f31f0400-9958-11eb-99ee-5073241bd668.png">

Btw, the [documentation](https://docs.commercetools.com/custom-applications/getting-started/installing-a-starter-application#installing-a-starter-application) does not mention to check out the repository.

**Comment by tomwscott** [comment]
Running:
``` 
npx @commercetools-frontend/create-mc-app \
  my-new-custom-application-project \
  --template starter
```
currently results in: 
```
Downloading template
    ✖ Cloning repository using branch master
      → fatal: Remote branch master not found in upstream origin
      Copying template starter into project directory /Users/tomwscott/my-new-custom-application-project
    Updating package.json
    Installing dependencies
Error: Command failed with exit code 128: git clone --branch=master --depth=1 https://github.com/commercetools/merchant-center-application-kit.git merchant-center-application-kit--master--1618250617302
Cloning into 'merchant-center-application-kit--master--1618250617302'...
warning: Could not find remote branch master to clone.
fatal: Remote branch master not found in upstream origin
    at makeError (/Users/tomwscott/node_modules/execa/lib/error.js:58:11)
    at handlePromise (/Users/tomwscott/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async Task.task (/Users/tomwscott/node_modules/@commercetools-frontend/create-mc-app/src/tasks/download-template.js:34:28) {
  shortMessage: 'Command failed with exit code 128: git clone --branch=master --depth=1 https://github.com/commercetools/merchant-center-application-kit.git merchant-center-application-kit--master--1618250617302',
  command: 'git clone --branch=master --depth=1 https://github.com/commercetools/merchant-center-application-kit.git merchant-center-application-kit--master--1618250617302',
  exitCode: 128,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: "Cloning into 'merchant-center-application-kit--master--1618250617302'...\n" +
    'warning: Could not find remote branch master to clone.\n' +
    'fatal: Remote branch master not found in upstream origin',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false,
  context: [Object: null prototype] {}
}
npm ERR! code 1
npm ERR! path /Users/tomwscott
npm ERR! command failed
npm ERR! command sh -c create-mc-app "my-new-custom-application-project" "--template" "starter"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tomwscott/.npm/_logs/2021-04-12T18_03_38_981Z-debug.log
```

It looks like it's still looking for the `master` branch which looks like it's been renamed to `main`

**Comment by emmenko** [comment]
Did you install the `@commercetools-frontend/create-mc-app` package globally? If so, please make sure you have at least version `18.2.3` installed (see [changelog](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/create-mc-app/CHANGELOG.md))

**Comment by tdeekens** [comment]
The usage if the master branch to be main was fixed in version 18.2.3 to my knowledge. I presume there is a cache or global package using the old version through npx.

**Comment by tomwscott** [comment]
Thanks, that worked a treat after tracking down the cached version.
